ballerina/tests/listener_test.bal

```diff
@@ -17,79 +17,81 @@
-//         json accountName = payload.changedData.get("Name");
-//         if (accountName.toString() == "WSO2 Inc") {
-//             lock {
-//                 isUpdated = true;
-//             }
-//             io:println("Updated " + payload.toString());
-//         } else {
-//             io:println(payload.toString());
-//         }
-//     }
-
-//     remote function onDelete(EventData payload) {
-//         string? eventType = payload.metadata?.changeType;
-//         if (eventType is string && eventType == "DELETE") {
-//             lock {
-//                 isDeleted = true;
-//             }
-//             io:println("Deleted " + payload.toString());
-//         } else {
-//             io:println(payload.toString());
-//         }
-//     }
-
-//     remote function onRestore(EventData payload) {
-//         string? eventType = payload.metadata?.changeType;
-//         if (eventType is string && eventType == "UNDELETE") {
-//             lock {
-//                 isRestored = true;
-//             }
-//             io:println("Restored " + payload.toString());
-//         } else {
-//             io:println(payload.toString());
-//         }
-//     }
-// }
+service "/data/ChangeEvents" on eventListener {
+    remote function onCreate(EventData payload) {
+        string? eventType = payload.metadata?.changeType;
+        if (eventType is string && eventType == "CREATE") {
+            lock {
+                isCreated = true;
+            }
+            io:println("Created " + payload.toString());
+        } else {
+            io:println(payload.toString());
+        }
+    }
+
+    remote isolated function onUpdate(EventData payload) {
+        json accountName = payload.changedData.get("Name");
+        if (accountName.toString() == "WSO2 Inc") {
+            lock {
+                isUpdated = true;
+            }
+            io:println("Updated " + payload.toString());
+        } else {
+            io:println(payload.toString());
+        }
+    }
+
+    remote function onDelete(EventData payload) {
+        string? eventType = payload.metadata?.changeType;
+        if (eventType is string && eventType == "DELETE") {
+            lock {
+                isDeleted = true;
+            }
+            io:println("Deleted " + payload.toString());
+        } else {
+            io:println(payload.toString());
+        }
+    }
+
+    remote function onRestore(EventData payload) {
+        string? eventType = payload.metadata?.changeType;
+        if (eventType is string && eventType == "UNDELETE") {
+            lock {
+                isRestored = true;
+            }
+            io:println("Restored " + payload.toString());
+        } else {
+            io:println(payload.toString());
+        }
+    }
+}
 
 // Using direct-token config for client configuration
 Client lisbaseClient = check new (sfConfigRefreshCodeFlow);
 string testRecordId = "";
 
 @test:Config {
-    enable: false
+    enable: true
```

Comment:
Do we need to manually do this ? I think this is enabled by default :thinking:

Reply:
no these are currently not running due to an auth error. 
Secret mixup which I sorted out with Maheshika. email " Adding GitHub Secrets to the Ballerina Salesforce Repository"